### PR TITLE
Toolbar Updates - Episode 4 (A New Hope)

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

There will be no new features here, just random bugfixes and small UX improvements that make a difference.
This should be merged in before the release.

### CODE REVIEW

**Comment by mariusandra** [comment]
@timgl ready for review & merge

**Comment by mariusandra** [comment]
I added a link [this issue](https://github.com/PostHog/posthog/issues/1129) to gather all toolbar feedback in one place... and pushed a few more cosmetic fixes since my last comment. Now ready to review!

File: frontend/src/toolbar/DockedToolbar.js [hunk 1]
```diff
@@ -7,17 +7,19 @@ import { ActionsTab } from '~/toolbar/actions/ActionsTab'
...
     const { hoverElement, selectedElement, inspectEnabled, heatmapEnabled } = useValues(elementsLogic)
     const { setSelectedElement } = useActions(elementsLogic)
+    const { dockTopMargin } = useValues(dockLogic)
 
     const showElementInsteadOfTabs =
         type === 'dock' && tab === 'stats' && (inspectEnabled || heatmapEnabled) && (hoverElement || selectedElement)
 
     return (
-        <div>
+        <div style={{ transform: `translate(0, ${dockTopMargin}px)`, transition: 'transform 0.3s' }}>
```
Comment (EDsCODE) [inline frontend/src/toolbar/DockedToolbar.js:22]: super smooth

**Review by EDsCODE — APPROVED** [review]
lgtm

**Pull request merged**